status-checker/cmd/checker/main.go

```diff
@@ -0,0 +1,270 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/x/consensus/types"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/health/grpc_health_v1"
+)
+
+// define grpc endpoints
+var (
+	// SSC endpoints
+	grpcMainnet = "ssc-grpc.sagarpc.io:9090"
+	grpcTestnet = "testnet2-grpc.sagarpc.io:9090"
```

Comment:
@luckychess why do you want to hard-code these endpoints? Can't they be read from the environment or some other config that does not need to be compiled and deployed each time an endpoint is added/updated/removed?